src/trading-api/next/v2/bullish-trading-api.yml

```diff
@@ -552,6 +584,10 @@ info:
     2. Sign the hexdigest from step 1 with ECDSA P-256 using your `<PRIVATE_KEY>`
     3. Various signing examples can be found on [GitHub](https://github.com/bullish-exchange)
 
+    ## Fetch Trading Account Ids
+    
+    Trading Account Ids may be required by some endpoints. They can be fetched from [Get All Trading Accounts](#get-/accounts/trading-accounts).
```

Comment:
added a section to fetch trading account ids.

![image](https://github.com/bullish-exchange/api-docs/assets/84704494/99522863-23f6-4dd5-943f-8fc58000b055)